src/index.js

```diff
@@ -107,6 +117,16 @@ app.post("/", async (req, res) => {
   }
 });
 
+const buildGetMetadataOptions = async (rpcUrl) => {
+  if (!rpcUrl) return;
```

Comment:
```suggestion
  if (!rpcUrl) {
    return undefined;
  }
```
Looks more explicit to me